package.json

```diff
@@ -8,8 +8,9 @@
     "build": "turbo run build",
     "reset": "./reset.sh",
     "test": "turbo run build --filter=testing && turbo run test",
-    "prettify": "prettier --write **/*.{js,ts,md}",
-    "prettify:watch": "onchange **/*.{js,ts,md} -- prettier --write {{changed}}",
+    "prettify": "prettier --write */**/**/*.{js,ts,md}",
```

Comment:
It would be safer to check only `docs/src`, `packages/**/src`, and `packages/**/README.md`.